### PR TITLE
Include CVE status in RSS feed

### PR DESCRIPTION
**Pull request by jbiers**
Includes the new CVE _status_ field for each item in the [RSS feed](https://kubernetes.io/docs/reference/issues-security/official-cve-feed/#cve-feeds-1).

Relies on changes from [kubernetes/sig-security#115](https://github.com/kubernetes/sig-security/pull/115)

### CODE REVIEW

**Comment by jbiers** [comment]
Change tested locally

**Review by dipesh-rawat — COMMENTED** [review]
Thanks!
[Preview RSS ](https://deploy-preview-47000--kubernetes-io-main-staging.netlify.app/docs/reference/issues-security/official-cve-feed/feed.xml)

/lgtm

File: layouts/_default/cve-feed.rss.xml [hunk 1]
```diff
@@ -42,6 +42,7 @@
       <pubDate>{{ time.Format "Mon, 02 Jan 2006 15:04:05 -0700" .date_published | safeHTML }}</pubDate>
       <guid>{{ .external_url }}</guid>
       <description>{{ htmlEscape .summary }}</description>
+      <status>{{ .status }}</status>
```
Comment (sftim) [inline layouts/_default/cve-feed.rss.xml:45]: There's no `<status>` element in Atom. Try:
```suggestion
      <category term="{{ .status }}" scheme="https://k8s.io/2024-06-27/cve-item-status" />
```

(I minted this scheme as an example)

Comment (jbiers) [reply layouts/_default/cve-feed.rss.xml:45]: Sure! I'm not sure I understand what the _scheme_ field should actually contain though. Would you mind clarifying that for me?

Comment (sftim) [reply layouts/_default/cve-feed.rss.xml:45]: It needs to be a URI, a bit like a relationship in the semantic web.https://graphdb.ontotext.com/documentation/9.5/free/introduction-to-semantic-web.html explains more but is complex to learn.

If you're not sure, go with the suggestion.

Comment (jbiers) [reply layouts/_default/cve-feed.rss.xml]: Done then :)

**Review by sftim — COMMENTED** [review]
/lgtm

/sig security

**Comment by katcosgrove** [comment]
/approve

Thanks!

**Pull request merged**